aries-site/src/pages/design/data-visualization.js

```diff
@@ -1,22 +1,25 @@
 import React from 'react';
 import { Text } from 'grommet';
 
-import { ContentSection, Layout, MainContent, SideBar } from '../../layouts';
+import {
+  ContentSection,
+  PageLayout,
+  MainContent,
+  SideBar,
+} from '../../layouts';
 import { MainHeading } from '../../components';
 
 const Index = () => (
-  <>
-    <Layout title="Data-visualization">
-      <SideBar>
-        <Text>Secondary Nav</Text>
-      </SideBar>
-      <MainContent>
-        <ContentSection>
-          <MainHeading>Data-visualization</MainHeading>
-        </ContentSection>
-      </MainContent>
-    </Layout>
-  </>
+  <PageLayout title="Data-visualization">
+    <SideBar>
```

Comment:
Some files still have the `SideBar` inside individual pages. I think we can remove it since it's now contained in `PageLayout` right?

Reply:
I'm on it, good catch

Reply:
I didn't comment on each file that has it, but there are a couple more in addition to this one.

Reply:
Of course. we should be good to go now.